src/bp/ui-studio/src/web/views/FlowBuilder/diagram/index.tsx

```diff
@@ -553,7 +553,13 @@ class Diagram extends Component<Props> {
     }
 
     if (selectedNode && (selectedNode.oldX !== selectedNode.x || selectedNode.oldY !== selectedNode.y)) {
-      this.props.updateFlowNode({ x: selectedNode.x, y: selectedNode.y })
+      const nodesToUpdate = []
```

Comment:
Is this condition only for the case where nodes have been moved? If so, can you rename this variable to something like nodesToMove?